extensions/azurePublish/src/node/luisAndQnA.ts

```diff
@@ -120,13 +120,13 @@ export async function publishLuisToPrediction(
       // DOCS HERE: https://westus.dev.cognitive.microsoft.com/docs/services/5890b47c39e2bb17b84a55ff/operations/5be313cec181ae720aa2b26c
       // This returns a list of azure account information objects with AzureSubscriptionID, ResourceGroup, AccountName for each.
       const getAccountUri = `${authoringEndpoint}/luis/api/v2.0/azureaccounts`;
-      const options = {
+      const options: AxiosRequestConfig = {
         headers: { Authorization: `Bearer ${accessToken}`, 'Ocp-Apim-Subscription-Key': luisAuthoringKey },
-      } as rp.RequestPromiseOptions;
-      const response = await rp.get(getAccountUri, options);
+      };
+      const response = await axios.get(getAccountUri, options);
 
       // this should include an array of account info objects
-      accountList = JSON.parse(response);
+      accountList = response?.data;
```

Comment:
should this be ```accountList = response?.data ?? [];```
if you are iterating on accountList and it's undefined, it will throw